tests/lib/Persistence/Legacy/Content/ContentHandlerTest.php

```diff
@@ -811,6 +811,37 @@ public function testUpdateMetadataUpdatesPathIdentificationString()
         );
     }
 
+    /**
+     * @throws \Ibexa\Core\Base\Exceptions\NotFoundException
+     */
+    public function testLoadRelation(): void
+    {
+        $handler = $this->getContentHandler();
+
+        $gatewayMock = $this->getGatewayMock();
+        $mapperMock = $this->getMapperMock();
+        $relationFixture = $this->getRelationFixture();
+
+        $gatewayMock
+            ->expects(self::once())
+            ->method('loadRelation')
+            ->with(self::equalTo(1))
```

Comment:
```suggestion
            ->with(1)
```

Unless you're using a different assertion (like `self::identicalTo()`), this is equivalent.